src/utils/stringUtils.ts

```diff
@@ -1,5 +1,9 @@
-export const flatJoin = (...classes: (string | string[])[]): string =>
-  classes.flat().join(' ');
+// Source https://emailregex.com/
+export const validateEmail = (email: string | undefined | null): boolean =>
+  !!email &&
+  /^(([^<>()[]\\.,;:\s@"]+(\.[^<>()[]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(
+    email,
+  );
 
-export const safeString = (value?: unknown): string =>
-  value === undefined || value === null ? '' : String(value).trim();
+export const validateTel = (tel: string | undefined | null): boolean =>
```

Comment:
Würde gerne Abkürzungen vermeiden: `validateTelephoneNumber`

Reply:
Erledigt in https://github.com/MORE-Platform/more-studymanager-frontend/pull/581/commits/0d3a3383a7da48d2b7ad39e22fda1a805cef6562